0001-interledger-architecture/0001-interledger-architecture.md

```diff
@@ -98,53 +98,62 @@ Ledger protocols are responsible for executing the individual transfers that con
 
 * Proposed -- Nothing has happened yet.
-* Prepared -- Funds are held in escrow.
+* Prepared -- Funds are held.
 * Executed -- The transfer has completed.
 * Rejected -- The transfer has been canceled (and funds returned to the sender.)
 
-<span class="show alert alert-info">**Hint:** Escrow is the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
+<span class="show alert alert-info">**Hint:** Authorization holds are the financial equivalent of a [two-phase commit](http://foldoc.org/two-phase%20commit).</span>
 
+The ledger MUST be able to release the held funds to the receiver upon receiving a 32-byte preimage whose SHA-256 hash matches a value provided by the sender. The ledger MUST accept ONLY 32 byte preimages or support specifying the fulfillment length when the transfer is prepared.
 
-The ledger MUST be able to verify a simple SHA-256 hashlock cryptographic escrow condition in order to automatically trigger the release of the escrow. It must also allow a timeout to automatically roll back an escrowed transfer.
+The ledger MUST support releasing held funds back to the sender after a timeout.
 
 The ledger MUST support attaching a short message or *memo* to each transfer.
 
+The ledger MUST support notifications to account holders when transfers are prepared, executed, or rejected that affect their accounts.
+
 #### Full Support
 
 The ledger MUST fulfill the requirements for basic support and also the following:
 
-It MUST support all cryptographic condition types with the status "recommended".
+The ledger MUST support memos up to 65535 bytes.
 
-It MUST support memos to be added for the credited and debited accounts separately and SHOULD support fairly large memo sizes.
+The ledger MUST support sending an authenticated message of up to 65535 bytes to the holder of another account on the ledger.
 
-It MUST support a way to discover connectors connected to the ledger.
+The ledger MUST support a way to look up a fulfillment by condition hash. It SHOULD automatically reject new transfers (that have not been prepared yet) that have an execution condition for which the ledger already knows the fulfillment. This aids in [error recovery](#error-recovery).
 
-It MUST support a way to look up a fulfillment by condition hash. It SHOULD automatically reject new transfers (that have not been prepared yet) that have an execution condition for which the ledger already knows the fulfillment. This aids in [UTP error recovery](#error-recovery).
+The ledger MUST support preparing, executing, and rejecting transfers in 1 second or less.
+
+The ledger MUST define an [ILP Address](../0015-ilp-addresses/0015-ilp-addresses.md) prefix and scheme such that accounts on the ledger can be addressed using canonical ILP addresses.
+
+The ledger MUST support off-ledger, ledger-secured payment channels.
```

Comment:
I think we should actually just take that out for now. The role of payment channels in ILP is something that we need to think more about and it's strange to call it a requirement for a ledger when we don't have a solid description of what exactly a ledger would need to do to fulfill that.